src/chains/ethereum/ethereum/tests/api/debug/debug_storageRangeAt.test.ts

```diff
@@ -40,92 +83,62 @@ describe("api", () => {
         deploymentBlockHash = deploymentTxReceipt.blockHash;
 
         methods = contract.contract.evm.methodIdentifiers;
-        const initialValue =
-          "0000000000000000000000000000000000000000000000000000000000000019";
-
-        const transactionHash = await provider.send("eth_sendTransaction", [
-          {
-            from: accounts[0],
-            to: contractAddress,
-            gas: "0x2fefd8",
-            data: `0x${methods["setValue(uint256)"]}${initialValue}`
-          }
-        ]);
+
+        const transactionHash = await updateValue("setValue(uint256)", "19");
         await provider.once("message");
 
-        const txReceipt = await provider.send("eth_getTransactionReceipt", [
+        ({ blockHash } = await provider.send("eth_getTransactionReceipt", [
           transactionHash
-        ]);
-        blockHash = txReceipt.blockHash;
+        ]));
 
         await provider.send("eth_unsubscribe", [subscriptionId]);
       });
 
-      after(async () => {
+      after("shut down provider", async () => {
         provider && (await provider.disconnect());
       });
 
       it("should return the storage for the given range", async () => {
-        const result = await provider.send("debug_storageRangeAt", [
-          blockHash,
-          0,
-          contractAddress,
-          "0x00",
-          2
-        ]);
+        const result = await debugStorage(blockHash, 0, 2);
 
         const storage = {
-          "0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563":
-            {
-              key: "0x0000000000000000000000000000000000000000000000000000000000000000",
-              value:
-                "0x0000000000000000000000000000000000000000000000000000000000000005"
-            },
-          "0x405787fa12a823e0f2b7631cc41b3ba8828b3321ca811111fa75cd3aa3bb5ace":
-            {
-              key: "0x0000000000000000000000000000000000000000000000000000000000000002",
-              value:
-                "0x68656c6c6f207270647200000000000000000000000000000000000000000014"
-            }
+          [VALUE1_KEY.hashKey]: {
+            key: VALUE1_KEY.key,
+            value:
+              "0x0000000000000000000000000000000000000000000000000000000000000005"
+          },
+          [VALUE3_KEY.hashKey]: {
+            key: VALUE3_KEY.key,
+            value:
+              "0x68656c6c6f20776f726c64000000000000000000000000000000000000000016"
```

Comment:
I updated the contract value from "hello rdpr" to "hello world", and this change reflects that.